dbt_sugar/core/connectors/postgres_connector.py

```diff
@@ -0,0 +1,55 @@
+"""
+Module Postgres connector.
+
+Module dependent of the base connector.
+"""
+from typing import List
+
+import sqlalchemy
+from base import BaseConnector
```

Comment:
I **think** (but I'm not sure) that we might want to use fully specified import paths here no? like `dbt_sugar.core.connectors.base` when we run and install this as a CLI app, but maybe we worry about this later.

I remember having had to do that when I was developing `sheetwork` so that's why I'm saying this, I might be totally wrong.

Reply:
done! I was testing in the folder and forgot to remove it.